### PR TITLE
Ygilquin media grid clean

### PR DESCRIPTION
**Pull request by ygilquin-gpsw**
Replace last pull request on ygilquin-media-grid

### CODE REVIEW

**Comment by sw-team-release-gpsw** [comment]
[CLA assistant check](https://cla.gopro.com/sign) <br/>All committers have signed the CLA.

**Comment by ygilquin-gpsw** [comment]
This PR can be tested by checking behaviour of "multiple media" sample project. Media grid project should be well displayed. Also on changing scene, background color should be displayed on the whole canvas instead of the last frame.

File: src/render/background/BackgroundMeshRenderer.js [hunk 1]
```diff
@@ -370,13 +391,91 @@ FORGE.BackgroundMeshRenderer.prototype._clear = function()
...
+    var quadri = new Int8Array(size);
+    var it = FORGE.Utils.arrayKeys(quadri);
+
+    var qa = new THREE.Vector2(1, 1);
+    var qb = new THREE.Vector2(1, -1);
+    var qc = new THREE.Vector2(-1, 1);
+    var qd = new THREE.Vector2(-1, -1);
+
+    var ipd = this._subdivision + 1; // indices per dimension
+    for (var f=0; f<6; f++)
```
Comment (aberthet-gpsw) [inline src/render/background/BackgroundMeshRenderer.js]: Spaces are missing, should be ` for (var f = 0; f < 6; f++)`. Same below.

File: src/render/view/ViewGoPro.js [hunk 2]
```diff
@@ -82,10 +81,21 @@ FORGE.ViewGoPro.prototype.updateUniforms = function(uniforms)
...
 
     var fovRad = 0.5 * FORGE.Math.degToRad(fov);
     this._projectionScale = (distance + 1) * Math.sin(fovRad) / (distance + Math.cos(fovRad));
+};
+
+/**
+ * Update uniforms.
+ *
+ * @method FORGE.ViewGoPro#updateUniforms
+ * @param {Object} uniforms
```
Comment (aberthet-gpsw) [inline src/render/view/ViewGoPro.js]: Missing extern `TUniform`

**Comment by aberthet-gpsw** [comment]
Test is good for me, cool to have these features ! 👍

**Pull request merged**